module/Database/src/Database/Form/MemberEdit.php

```diff
@@ -150,6 +158,12 @@ public function getInputFilterSpecification()
                 'validators' => array(
                     array('name' => 'digits')
                 )
+            ),
+            'tuenumber' => array(
+                'required' => false,
+                'validators' => array(
+                    array('name' => 'digits')
+                )
```

Comment:
This was just copied from the subscription form:

https://github.com/GEWIS/gewisdb/blob/6e7ca59e3fd6292a9c4e7a09b103e34a2e9ef4dd/module/Database/src/Database/Form/Member.php#L292-L297

Since it is just a simple regex I will change both occurrences to use the ZF regex validator (will look like this):

```php
'tueusername' => array(
    'required' => false,
    'validators' => array(
        array(
            'name' => 'regex',
            'options' => array(
                'pattern' => '/^(s\d{6}|\d{8})$/',
            ),
        ),
    ),
),
```